product_docs/docs/pgd/5/durability/commit-scope-rules.mdx

```diff
@@ -2,49 +2,49 @@
 title: Commit scope rules
 ---
 
-Commit scope rules are at the core of the commit scope mechanism. They define what the commit scope will enforce. 
+Commit scope rules are at the core of the commit scope mechanism. They define what the commit scope enforces. 
 
-Commit scope rules are composed of one or more operations combined with an `AND`. 
+Commit scope rules are composed of one or more operations connected by an `AND`. 
 
-Each operation is made up of two (or three) parts, the commit scope group, an optional confirmation level, and the kind of commit scope (which may have it's own parameters). 
+Each operation is made up of two or three parts: the commit scope group, an optional confirmation level, and the kind of commit scope, which can have its own parameters.
 
 ```
-commit_scope_group [ confimation_level ] commit_scope_kind
+commit_scope_group [ confirmation_level ] commit_scope_kind
 ```
 
-A full formal syntax diagram is available in the [commit scope reference section.](/pgd/latest/reference/commit-scopes/#commit-scope-syntax).
+A full formal syntax diagram is available in the [Commit scopes](/pgd/latest/reference/commit-scopes/#commit-scope-syntax) reference.
 
-If we look at a typical commit scope rule, we can now break it down into its components:
+This typical commit scope rule can be broken down into its components:
```

Comment:
These edits...I was trying to follow the logic. The use of the word "here" was confusing me, and I ultimately decided it all meant to refer to the same example. Let me know if I get any of what was meant here wrong and we can revisit.

Reply:
```suggestion
A typical commit scope rule, such as `ANY 2 (group) GROUP COMMIT`, can be broken down into its components:
```

Maybe clearer - just lose the code block below.

Oh and the "here" is something I completely forgot as this content started as a conversational examination of scopes.